hmm.py

```diff
@@ -1,39 +1,61 @@
 import re
-import subprocess
+import subprocess, shlex
 import tempfile
+import os
+from os import mkdir
 from typing import List
 
 from config import UNIPROT_DATABASE
-from constants import REGEX_NAME
+from constant import REGEX_NAME, REGEX_PROTEIN
 
 
 def split_models(models_file: str) -> List[str]:
     """Get one model files, split in one file per model and returns the model names."""
     models = []
+    folder_data_models_path = "data/models_hmm"
+    if not os.path.exists(folder_data_models_path):
+        mkdir("data")
+        mkdir(folder_data_models_path)
```

Comment:
This should be two `if` one for the `data` dir and another one for the `models` dir